scripts/collect_workloads.py

```
@@ -1015,6 +949,111 @@ def _pad_to_tokens(msgs: list[dict], approx_tokens: int) -> list[dict]:
+                    else:
+                        passed += 1
+                except Exception:
+                    pass
+
+            # Copy to per-def baseline trace name
+            import shutil
+
+            shutil.copy(auto_trace, out_trace)
+            print(f"  Results: {passed} PASSED, {len(failed)} FAILED")
+            if failed:
+                print(f"  FAILED workloads:")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)